providers/hetznercloud/hcapi/mocks/mock_Client.go

```diff
@@ -0,0 +1,317 @@
+// Code generated by mockery. DO NOT EDIT.
```

Comment:
Out of curiosity: why is it generating several files here? Is this using some different  compared to the main repo?

Reply:
I think that can be configured, I can try to generate only one file.

Reply:
The usage of `//go:generate mockery ...` is deprecated. I'm using the recommended `.mockery.yaml` config file and I don't see a way to generate multiple interfaces into a single file.